packages/components/pie-card-container/src/defs.ts

```diff
@@ -1,9 +1,14 @@
 export const variants = ['default', 'outline', 'inverse', 'outline-inverse'] as const;
 export const interactionTypes = ['anchor', 'button', 'none'] as const;
-export const padding = ['a', 'b', 'c', 'd', 'e', 'f', 'g'] as const;
-
-export type Variant = typeof variants[number];
-export type PaddingValue = typeof padding[number];
+export const paddingValues = [
```

Comment:
More of a suggestion so it might be easier to maintain in future (incase we add further values). You could define a set of the initial padding values i.e `'a-g'` in a defined const `paddingValues` then push the combination into it or just export the original like you have here but use a bit of functionality inside. Something like below:
Happy to keep it like you have it though so just a suggestion.

```javascript
export const paddingValues = (() => {
  const variants = ['a', 'b', 'c', 'd', 'e', 'f', 'g'];
  const result: (string)[] = [];

  variants.forEach((first) => {
    result.push(first);
    variants.forEach((second) => {
      result.push(`${first},${second}`);
    });
  });

  return result as const;
})();
```

Reply:
good idea, i refactored to that but with a slightly different approach 🤝